textual/src/toga_textual/app.py

```diff
@@ -74,6 +74,13 @@ def set_main_window(self, window):
     def get_screens(self):
         return [ScreenImpl(window._impl.native) for window in self.interface.windows]
 
+    ######################################################################
+    # App state
+    ######################################################################
+
+    def get_dark_mode_state(self):
+        return None
```

Comment:
[Textual has the ability to differentiate between light and dark themes](https://textual.textualize.io/guide/design/#light-and-dark-themes), so this should probably be handled the same as other platforms.